pkg/loki/modules.go

```diff
@@ -450,6 +450,13 @@ func (t *Loki) initQuerier() (services.Service, error) {
 func (t *Loki) initIngester() (_ services.Service, err error) {
 	t.Cfg.Ingester.LifecyclerConfig.ListenPort = t.Cfg.Server.GRPCListenPort
 
+	if t.Cfg.Ingester.ShutdownMarkerPath == "" && t.Cfg.Common.PathPrefix != "" {
```

Comment:
could you check if this works with SSD?

Reply:
Yes it works with the `write` path of SSD. I get the 500 error when the paths are not set. The warning also shows up in the logs in that case. With common.path_prefix set it works as expected.